File: FloridaShiduch/Scripts/src/models/bindings/demographics.js
define(['bindings', 'behaviors/demographics'], function (Bindings, DemographicsBehavior) {
    'use strict';
    // Singleton
    var bindings = new Bindings({
        '@ui.firstName': {
            observe: 'firstName',
            setOps: { validate: true }
        },
        '@ui.lastName': {
            observe: 'lastName',
            setOps: { validate: true }
        },
        '@ui.gender': {
            observe: 'gender',
            setOps: { validate: true }
        },
        '@ui.address': {
            observe: 'address',
            setOps: { validate: true }
        },
        '@ui.apartment': {
            observe: 'apartment',
            setOps: { validate: true }
        },
        '@ui.city': {
            observe: 'city',
            setOps: { validate: true }
        },
        '@ui.state': {
            observe: 'state',
            setOps: { validate: true }
        },
        '@ui.zip': {
            observe: 'zip',
            setOps: { validate: true }
        },
        //countryId: 'countryId',
        '@ui.homePhone': {
            observe: 'homePhone',
            setOps: { validate: true }
        },
        '@ui.workPhone': {
            observe: 'workPhone',
            setOps: { validate: true }
        },
        '@ui.mobilePhone': {
            observe: 'mobilePhone',
            setOps: { validate: true }
        }
    }, DemographicsBehavior.prototype.ui);

    return bindings;
});

File: FloridaShiduch/Models/Profile/Demographics.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Data.Entity;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Newtonsoft.Json;

namespace FloridaShiduch.Models.Profile
{
    public class Demographic
    {
        [Key, ForeignKey("ApplicationUser")]
        [JsonProperty("userId")]
        public string UserId { get; set; }

        [JsonProperty("firstName")]
        public string FirstName { get; set; }

        [JsonProperty("lastName")]
        public string LastName { get; set; }

        [JsonProperty("gender")]
        public string Gender { get; set; }

        [JsonProperty("address")]
        public string Address { get; set; }

        [JsonProperty("apartment")]
        public string Apartment { get; set; }

        [JsonProperty("city")]
        public string City { get; set; }

        [JsonProperty("state")]
        public string State { get; set; }

        [JsonProperty("zip")]
        public string Zip { get; set; }

        [JsonProperty("countryId")]
        public int CountryId { get; set; }

        [JsonProperty("homePhone")]
        public string HomePhone { get; set; }

        [JsonProperty("workPhone")]
        public string WorkPhone { get; set; }

        [JsonProperty("mobilePhone")]
        public string MobilePhone { get; set; }

        public virtual ApplicationUser ApplicationUser { get; set; }
    }
}